src/main/java/Chat/tasks/Evelyn.java

```diff
@@ -0,0 +1,261 @@
+package Chat.tasks;
+
+import Chat.exceptions.RepeatMark;
+import Chat.exceptions.RepeatUnmark;
+import Chat.exceptions.InvalidIndex;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import java.util.ArrayList;
+import java.io.File;
+
+
+public class Evelyn {
+    public static ArrayList<Task> tasks;
+    public static int indexOfTask = 0;
+
+    public static void markTask(int index, boolean done) throws InvalidIndex, RepeatMark, RepeatUnmark {
+        if(index < 0 || index >= indexOfTask){
+            throw new InvalidIndex();
+        }
+
+        if (done) {
+            try {
+                tasks.get(index).markAsDone();
+                System.out.println("Nice! I've marked this task as done:");
+            } catch(RepeatMark e){
+                System.out.println("This task is already marked as done.");
+            }
+        } else {
+            try {
+                tasks.get(index).markAsNotDone();
+                System.out.println("OK, I've marked this task as not done yet:");
+            } catch(RepeatUnmark e){
+                System.out.println("This task is already unmarked as not done.");
+            }
+        }
+        System.out.println("  " + tasks.get(index).getStatusIcon() + " "
+                               + tasks.get(index).description);
+        printLine();
+
+    }
+
+    public static void printList(ArrayList<Task> tasks){
```

Comment:
```suggestion
    public static void printList(ArrayList<Task> tasks) {
```

Take note of the form of the funtion declaration. There should be a space before the {